pkg/cluster/base.go

```diff
@@ -145,6 +145,18 @@ func (p *ProviderBase) GetClusterOptions() []types.Flag {
 			V:     p.Mirror,
 			Usage: "For Chinese users, set INSTALL_K3S_MIRROR=cn to use the mirror address to accelerate k3s binary file download",
 		},
+		{
+			Name:  "docker-arg",
+			P:     &p.DockerArg,
+			V:     p.DockerArg,
+			Usage: "Parameter for docker script",
```

Comment:
It's better to give an example of how to set args. 
`Parameters for docker install script, e.g. xxxxxx`